airdrop/airdrop-distributor/src/main.sw

```diff
@@ -0,0 +1,100 @@
+contract;
+
+dep events;
+dep errors;
+dep data_structures;
+dep interface;
+dep utils;
+
+use events::{ClaimEvent, CreateAirdropEvent};
+use errors::{AccessError, InitError, StateError, VerificationError};
+use data_structures::ClaimData;
+use interface::AirdropDistributor;
+use std::{
+    address::Address,
+    block::height,
+    contract_id::ContractId,
+    hash::sha256,
+    identity::Identity,
+    logging::log,
+    revert::require,
+    storage::StorageMap,
+};
+use sway_libs::binary_merkle_proof::{leaf_digest, verify_proof};
+use utils::mint_to;
+
+storage {
+    /// Stores the ClaimData struct of users that have interacted with the Airdrop Distrubutor contract.
+    /// Maps (user => claim)
+    claims: StorageMap<Identity, ClaimData> = StorageMap {},
+    /// The block at which the claiming period will end.
+    end_block: u64 = 0,
+    /// The computed merkle root which is to be verified against.
+    merkle_root: b256 = 0x0000000000000000000000000000000000000000000000000000000000000000,
```

Comment:
This should be an option in the future when the SDK supports it.
I thought I posted it somewhere else too but I cannot find it now... I think any default value should eventually be set to None from Option. This will prevent any potential weird behaviour if the value turns out to be valid in some condition